tests/acceptance/servicecatalog/helpers_test.go

```diff
@@ -62,6 +62,24 @@ func awaitCatalogContainsServiceClasses(t *testing.T, timeout time.Duration, ser
 	}
 }
 
+// awaitBrokerServiceClass awaits for service broker's not empty catalog response.
```

Comment:
can you try to use wait.AtMost function to be more consistent? Please remember that AtMost has a small bug, so ask @mszostok to copy fix from him. Additionally, maybe renaming will be required: wait.ForFuncAtMost?

Reply:
Ok, I will sync with @mszostok and align this solution to use shared waiter.